### PR TITLE
Added io.js 2.0.0 support

### PR DESCRIPTION
**Pull request by jwilsson**
I haven't made any changes since all the tests passed.

I just added a new file and version. Let me know if you want a different name on the file!

### CODE REVIEW

**Comment by jwilsson** [comment]
I doesn't look like AppVeyor has updated to io.js 2.x yet (I noticed the error on Travis for one of my projects in the first place) and I assume we want to test on both io.js 1.x and 2.x but I have yet to find a way to specify both versions for AppVeyor.

Perhaps it's as simple as

    - nodejs_version: "1.0"
    - nodejs_version: "2.0"

**Comment by blond** [comment]
> Perhaps something as simple as
```
- nodejs_version: "1.0"
- nodejs_version: "2.0"
```

@jwilsson should work. Usually in AppVeyor a new version of the `Node.js` or `io.js` does not appear very quickly.

**Comment by blond** [comment]
Now the tests in Travis [were run](https://travis-ci.org/tschaub/mock-fs/jobs/61274324) only for the `2.x` version. 

```
...
Now using io.js v2.0.0
$ node --version
v2.0.0
...
```

@jwilsson You need to add run tests in `iojs@1.x` and `iojs@2.x` for Travis.

```
node_js:
  - iojs-v1
  - iojs-v2
```

**Comment by jwilsson** [comment]
Ah, my bad!

Both io.js 1.x and 2.x are added in the respective `.yml` file but I guess we'll have to wait for AppVeyor then.

**Comment by programble** [comment]
+1

File: appveyor.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ environment:
     - nodejs_version: "0.8"
     - nodejs_version: "0.10"
     - nodejs_version: "1.0"
+    - nodejs_version: "2.0"
```
Comment (tschaub) [inline appveyor.yml]: How about removing this until AppVeyor supports 2.0?

Comment (jwilsson) [inline appveyor.yml]: Sure thing!

**Comment by tschaub** [comment]
Thanks for this @jwilsson.

As a side note, I'd like to rework things to avoid copying in the `fs` module for every version.  We should be able to use `process.binding('natives').fs` instead - but there have been some version specific workarounds.  If anybody is game to experiment with alternatives, please jump in.

**Pull request merged**